files/fr/web/html/link_types/manifest/index.html

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Types de liens : manifest'
+slug: Web/HTML/Link_types/manifest
+browser-compat: html.elements.link.rel.manifest
+translation_of: 'Web/HTML/Link_types/manifest'
+---
+<p><span class="seoSummary">Le mot-clé <strong><code>manifest</code></strong> de l'attribut {{HTMLAttrxRef("rel", "link")}} de l'élément {{HTMLElement("link")}} est une information indiquant que la ressource ciblée est un <a href="/fr/docs/Web/Manifest">manifeste d'application web</a>.</span></p>
```

Comment:
Same as https://github.com/mdn/translated-content/pull/1794/files#r679360769

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1795/commits/ea3ca056c9a2c1b866be31ae27d6fce492ab0c81